tests/test_json_parsing.py

```diff
@@ -0,0 +1,24 @@
+"""
+Tests for JSON parsing error handling bugs in server endpoints.
+"""
+
+
+def test_json_parsing_errors_in_close_endpoint(client):
```

Comment:
I guess this file becomes superfluous is `/close` takes no body.

Reply:
Removed this test